ios/FluentUI/Core/Theme/Tokens/AliasTokens.swift

```diff
@@ -657,69 +657,81 @@ public final class AliasTokens {
     // MARK: Shared Colors
 
     public enum SharedColorsTokens: CaseIterable {
-        // Red (Danger)
-        case redBackground1
-        case redBackground3
-        case redForeground1
-        case redForeground3
-        // Dark orange (Severe)
-        case darkOrangeBackground1
-        case darkOrangeBackground3
-        case darkOrangeForeground1
-        // Yellow (Warning)
-        case yellowBackground1
-        case yellowBackground3
-        case yellowForeground1
-        // Green (Success)
-        case greenBackground1
-        case greenBackground2
-        case greenForeground1
+        // Danger (Red)
```

Comment:
Was this order decided by the designers? Or could we switch Warning and Success to make it alphabetical?

Reply:
Though I guess right now they're in order of most concerning to least

Reply:
That order also sounds good. It made a bit more sense when we had 30 instead of 4, but it works.